src/utils/ucx/ucx_utils.cpp

```diff
@@ -236,11 +236,25 @@ nixl_status_t nixlUcxEp::disconnect_nb()
  * Active message handling
  * =========================================== */
 
-nixl_status_t nixlUcxEp::sendAm(unsigned msg_id,
-                                void* hdr, size_t hdr_len,
-                                void* buffer, size_t len,
-                                uint32_t flags, nixlUcxReq &req)
-{
+using nixl_ucx_am_cb_ctx_t = std::pair<void *, nixlUcxEp::am_deleter_t>;
+using nixl_ucx_am_cb_ctx_ptr_t = std::unique_ptr<nixl_ucx_am_cb_ctx_t>;
+
+void
+nixlUcxEp::sendAmCallback(void *request, ucs_status_t status, void *user_data) {
```

Comment:
This method can be put into unnamed namespace in .cpp file to avoid adding it to the class interface.